nuxeo-coldstorage/src/main/resources/OSGI-INF/coldstorage-bulk-contrib.xml

```diff
@@ -33,6 +33,7 @@
     </streamProcessor>
     <streamProcessor name="checkColdStorageAvailability"
       class="org.nuxeo.coldstorage.action.CheckColdStorageAvailabilityAction"
+      defaultScroller="${nuxeo.bulk.action.checkColdStorageAvailability.scroller:=default}"
```

Comment:
The prop should also be defined in the template nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nuxeo.defaults